data_safe_haven/config/config_sections.py

```diff
@@ -67,7 +68,7 @@ class ConfigSectionSRE(BaseModel, validate_assignment=True):
     @field_validator(
         "admin_ip_addresses",
         "data_provider_ip_addresses",
-        "research_user_ip_addresses",
+        # "research_user_ip_addresses",
```

Comment:
Yes, it was deliberately only the research users as that seemed to be the use case.

Exposing the admin filter to the Internet feels much riskier.
I would be inclined to wait and see if it is a feature users want.

Reply:
Agreed that admins might be worth holding off. Data providers is definitely needed - it's really common that they don't have a fixed IP address (or don't know what their assigned range is if they do).

Reply:
`curl https://ipinfo.io` 🙄.

OK, let's make that change. Shall we do that as a new PR into the rc branch? I think it will be clearer where it comes from rather than hiding it in this merge branch that way.

Reply:
Yes, I think that's a good idea

Reply:
It is in #2247